src/utils/beacon/decodePayload.test.ts

```diff
@@ -0,0 +1,37 @@
+import { decodePayload } from "./decodePayload";
+
+describe("decodePayload", () => {
+  it("should decode a valid payload with '0501' prefix and padding", () => {
```

Comment:
`it("should decode ..."` is always longer than `it("decodes ...` so it's better to avoid the word should unless really needed